apps/website/app/page.tsx

```diff
@@ -461,6 +450,51 @@ export default function Home() {
             </CardContent>
           </Card>
 
+          {/* Blog Section */}
```

Comment:
for Production we should probably add something like

```{blogs.length > 0 && (```

and delete the example posts